README.md

```diff
@@ -90,14 +97,27 @@ circuit_handler do |handler|
  handler.invocation_timeout = 10
  handler.excluded_exceptions += [NotConsideredFailureException]
 end
+
+# An example of a make request
+multi = Px::Service::Client::Multiplexer.new
+method = :get
+
+req = subject.make_request(method, url)
+multi.context do
+   multi.do(req)
+end.run
 ```
 
-Adds a circuit breaker to the client.  The circuit will open on any exception from the wrapped method, or if the request runs for longer than the `invocation_timeout`.
+Adds a circuit breaker to the client.  If the underlying method succeeds, `make_request` returns `Px::Service::Client::RetriableResponseFuture`
```

Comment:
it ALWAYS returns a future.  Sometimes the future will raise an error, that's all.